skrf/media/media.py

```diff
@@ -159,21 +162,21 @@ def npoints(self, val):
         self.frequency.npoints = val
 
     @property
-    def z0(self) -> npy.ndarray:
+    def z0_port(self) -> npy.ndarray:
         """
         Characteristic Impedance.
 
         Returns
         -------
         z0 : :class:`numpy.ndarray`
         """
```

Comment:
I think additional explanation should be given about what is returned here (None or something)

Reply:
Now that `z0_override` is a property of the abstract class `Media`, the switch between `z0_override` and abstract method `z0_characteristic` can be done here. The media classes will need to define z0_characteristic, which sounds quite clear what it is 😉 
```Python
    @property
    @abstractmethod
    def z0_characteristic(self):
        """
        Characteristic Impedance.
        This abstract method has to be defined in the Media Class.
        
        Returns
        -------
        z0_characteristic: npy.ndarray
            Characteristic Impedance in units of ohms
        """
        return None
    
    @property
    def z0(self):
        """
        Return Characteristic Impedance `z0_characteristic`.
        If `z0_override` is not None, it is returned instead.
        
        Returns
        -------
        z0 : npy.ndarray
            z0_characteristic or z0_override in units of ohms
        """
        if self.z0_override is None:
            return self.z0_characteristic
        else:
            return self.z0_override
```

Reply:
Done.